django-stubs/contrib/auth/backends.pyi

```diff
@@ -35,6 +35,6 @@ class AllowAllUsersModelBackend(ModelBackend): ...
 class RemoteUserBackend(ModelBackend):
     create_unknown_user: bool = ...
     def clean_username(self, username: str) -> str: ...
-    def configure_user(self, user: User) -> User: ...
+    def configure_user(self, request: Any, user: User) -> User: ...
```

Comment:
I guess it should be `request: HttpRequest`

Reply:
The `request` is passed from `authenticate()` method and the `request` type in that function is `Any`

https://github.com/typeddjango/django-stubs/pull/532/files#diff-d3e7fda227dc43ac762742004613707006b194955b72f3ec448f41132956de56R14

Reply:
Any ideas what else can it be?

Reply:
No idea. I just said I'm not sure while I didn't wrote the definition of `authenticate()`.

Reply:
let's change both then!

Reply:
@sobolevn Done